src/Microsoft.CSharp/tests/Microsoft.CSharp.Tests.csproj

```diff
@@ -12,7 +12,7 @@
     <Compile Include="CSharpArgumentInfoTests.cs" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\src\Microsoft.CSharp.csproj">
+    <ProjectReference Include="..\pkg\Microsoft.CSharp.pkgproj">
```

Comment:
Have we looked at the VS experience after switching to pkgproj? Can I still load up an individual library solution and make a change in the library code and it get property rebuilt when just rebuilding and running the test project?